### PR TITLE
Avoid nft list refresh animation on automatic refetch

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-490](https://emurgo.atlassian.net/browse/YOMO-490)

[YOMO-490]: https://emurgo.atlassian.net/browse/YOMO-490?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by banklesss** [comment]
The initial loading is covered by this https://github.com/Emurgo/yoroi-mobile/blob/7ee8c6e19cf0ee2e7ab2364062c27dda87a901fd/src/Nfts/Nfts.tsx#L32

Skeleton

File: src/Nfts/Nfts.tsx [hunk 1]
```diff
@@ -10,15 +10,21 @@ import {useNavigateTo} from './navigation'
 import {NoNftsScreen} from './NoNftsScreen'
 
 export const Nfts = () => {
-  const {search, nfts, isLoading, refetch, isRefetching, isError} = useFilteredNfts()
+  const {search, nfts, isLoading, refetch, isError} = useFilteredNfts()
```
Comment (stackchain) [inline src/Nfts/Nfts.tsx]: `isRefetching` | `isFecthing` wouldn't work here?

Comment (banklesss) [reply src/Nfts/Nfts.tsx]: both have the same values when using `refetch` for manual and automatic actions, so it is not possible to differentiate the origin of the action

Comment (banklesss) [reply src/Nfts/Nfts.tsx]: according to requirements we need to discriminate the animation when the action is automatic

**Review by stackchain — APPROVED** [review]

File: src/Nfts/Nfts.tsx [hunk 2]
```diff
@@ -10,15 +10,21 @@ import {useNavigateTo} from './navigation'
...
   const navigateTo = useNavigateTo()
   const handleNftSelect = (index: number) => navigateTo.nftDetails(nfts[index].id)
+  const [isRefreshing, setIsRefreshing] = React.useState(false)
   const strings = useStrings()
 
+  const onRefresh = React.useCallback(() => {
+    setIsRefreshing(true)
+    refetch().then(() => setIsRefreshing(false))
```
Comment (stackchain) [inline src/Nfts/Nfts.tsx]: OnSettled

**Review by stackchain — APPROVED** [review]

**Pull request merged**